rust/src/wallet.rs

```diff
@@ -182,6 +195,48 @@ impl Wallet {
     pub fn get_node_id(&self) -> PublicKey {
         self.lightning.channel_manager.get_our_node_id()
     }
+
+    pub fn get_script_status(&self, script: Script, txid: Txid) -> Result<ScriptStatus> {
+        let script_status = self
+            .lightning
+            .wallet
+            .get_tx_status_for_script(script, txid)
+            .map_err(|_| anyhow!("Could not get tx status for script"))?;
+        Ok(script_status)
+    }
+
+    pub fn send_to_address(&self, send_to: Address, amount: u64) -> Result<Txid> {
+        let wallet = self.lightning.wallet.get_wallet();
```

Comment:
❓ Shouldn't we sync with the blockchain like we do in ItchySats?

Reply:
I don't remember why this was needed, but I'm happy to add it!